ci/e2e-tests.sh

```diff
@@ -358,6 +358,13 @@ trap "
                         --resource-group "$AZURE_RESOURCE_GROUP_NAME" --name "$common_resource_name" \
                         --query 'location' --output tsv
                 )"
+
+            # For some reason, `az resource tag` must come after
```

Comment:
Well, it's more like:

```
# `az iot dps linked-hub create` deletes the tags on the DPS that were set by `az iot dps create`
# for some unknown reason, so we need to tag it again.
```

Also, please restore the `--tags` parameter of the initial `az iot dps create` so that the DPS is tagged in case the script fails at the `linked-hub create` step.

Reply:
dangit, my timing \*shakes fist\*

Should I open a hotfix PR for this, or can I lump it in with my upcoming dps-symkey PR?

Reply:
Latter is fine. No big deal. It's not like e2e tests are running automatically anyway.